Update README.rst, add some more details ad links to conform to iReal… (#3049)
minvws/nl-kat-coordination: README.rst

- `OpenKAT group on Linkedin <https://www.linkedin.com/>`_ (search for OpenKAT)
- IRC: #openkat on irc.libera.chat
- `Signal group <https://signal.group/#CjQKIIS4T1mDK1RcTqelkv-vDvnzrsU4b2qGj3xIPPrqWO8HEhDISi92dF_m4g7tXEB_QwN_>`_

Privacy
=======

> OpenKAT is not designed to collect private information, and in does not act on Private information that its finds. Some information considered to be Identifying information Might be collected through one or more of OpenKAT's plugins and subsequently stored, but only if that information was accessible to OpenKAT. Eg, a phone number or emailadres listed on a website might end up being collected in the 'proofs' that OpenKAT collects of It's actions. No Email or Phone number Models are present and as such they won't be processed into Objects by OpenKAT.

Reviewer: ```suggestion
OpenKAT is not designed to collect private information and it does not act on any private information that its finds. Some information considered to be personally identifying information might be collected through one or more of OpenKAT's plugins and subsequently stored, but only if that information was accessible to OpenKAT. E.g. a phone number or email address listed on a website might end up being collected as part of OpenKAT normal data collection. This data might then be stored for a long time because OpenKAT stores 'proofs' of its actions. No email or phone number models are present and as such they won't be processed into objects by OpenKAT.
```

> An OpenKAT installation requires user accounts for users to be able to login. These accounts (as all data OpenKAT works with) are stored only on the OpenKAT installation itself, and are not shared with other parties or outside of your OpenKAT install.

Reviewer: ```suggestion
An OpenKAT installation requires user accounts for users to be able to login. These accounts (and all data OpenKAT works with) are stored only on the OpenKAT installation itself and are not shared with other parties or outside of your OpenKAT install.
```

Security
========

> OpenKAT is designed to be as secure as possibly, however the default setup for development has some debugging flags enabled, and does not include TLS out of the box. To setup a secure production OpenKAT install, please follow the `Production setup guidelines <https://docs.openkat.nl/installation_and_deployment/install.html#production-environments>`_ and `Hardening guidelines <https://docs.openkat.nl/installation_and_deployment/hardening.html>`_.

Reviewer: Nitpicking here, but security is always a trade-off, so I wouldn't say it is "as secure as possible".

```suggestion
OpenKAT is designed to be secure by default in its production setup. In the development setup some debugging flags are enabled by default and it will not include TLS out of the box. To setup a secure production OpenKAT install, please follow the `Production setup guidelines <https://docs.openkat.nl/installation_and_deployment/install.html#production-environments>`_ and `Hardening guidelines <https://docs.openkat.nl/installation_and_deployment/hardening.html>`_.
```
